yt_dlp/extractor/patreon.py

```diff
@@ -26,12 +54,15 @@ class PatreonIE(InfoExtractor):
             'id': '743933',
             'ext': 'mp3',
             'title': 'Episode 166: David Smalley of Dogma Debate',
-            'description': 'md5:713b08b772cd6271b9f3906683cfacdf',
+            'description': 'md5:34d207dd29aa90e24f1b3f58841b81c7',
```

Comment:
Do make sure that the md5 changed because of actual change in site and not because the description extraction is now broken